diesel/src/sqlite/connection/functions.rs

```diff
@@ -76,6 +62,35 @@ where
     Ok(())
 }
 
+pub(crate) fn build_sql_function_args<ArgsSqlType, Args>(args: &[*mut ffi::sqlite3_value]) -> Result<Args, Error>
```

Comment:
Can we use these functions in the existing non aggregate versions too?

Reply:
Are you referring to `register()` in `src/sqlite/connection/functions.rs` or is there another place I am missing?

Reply:
Oh, I've missed that `register` already uses this function, so forget what I've wrote above 🤦‍♂